scripts/systemtap/kem/kem_run.sh

```diff
@@ -70,14 +70,14 @@ sleep 2
 
 for i in $(seq 0 $(($(nproc)-1)))
 do
-    rm -f /dev/kemd$i
+    rm -f /dev/kemd"$i"
 done
 
 echo Removing kemd.ko
-rmmod $KEM_DIR/kemd.ko
+rmmod "$KEM_DIR"/kemd.ko
 
 for i in $(seq 0 $(($(nproc)-1)))
 do
-    $KEM_DIR/../../../utils/m0run m0addb2dump $PWD/_kemc$i/o/100000000000000:2 | grep pagefault -A 1 | head -n 40
-    $KEM_DIR/../../../utils/m0run m0addb2dump $PWD/_kemc$i/o/100000000000000:2 | grep ctx_switch -A 1 | head -n 40
+    "$KEM_DIR"/../../../utils/m0run m0addb2dump "$PWD"/_kemc"$i"/o/100000000000000:2 | grep pagefault -A 1 | head -n 40
```

Comment:
The same question as the previous one.
Do you prefer:
"$KEM_DIR/../../../utils/m0run m0addb2dump" "$PWD/_kemc$i/o/100000000000000:2"

Reply:
done